tests/OBSLocal.py

```diff
@@ -217,6 +218,8 @@ def __init__(self, project=PROJECT):
         self.setup_remote_config()
         self.load_config()
         self.api = StagingAPI(APIURL, project)
+        # The ProductVersion is required for some actions, for example, when a request is accepted
+        self.create_attribute_type('OSRT', 'ProductVersion', 1)
```

Comment:
Doesn't this belong to `setup_remote_config`? Just like the call to `self.create_attribute_type('OSRT', 'Config', 1)`. Isn't it?

Reply:
Moreover, I think that doing this after the call to `load_config` is specially bad, since it guarantees `self.config` is outdated in comparison to what the OBS instance holds. Or am I lost?

Reply:
Hmmm, not really. `load_config` loads the value of the attribute `Config` associated to the target project. That has nothing to do with the existence of an attribute type called `ProductVersion`.

As far as I understand, attributes are associated to concrete projects and attribute types are a global thing that defines which attributes can exist. So there is no direct relationship between `self.config' and the attribute types beyond `Config` itself.

So maybe my fault, but calling `self.attributes` to the attribute holding the list of attribute types (not to be confused with real attributes) didn't make it any easier to understand.

Reply:
I tried to address that confusion as part of https://github.com/openSUSE/openSUSE-release-tools/pull/2626